k8s-manifests/database/deployment.yml

```
@@ -0,0 +1,52 @@
+ labels:
+   app: mongodb
+
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: mongodb
+
+  template:
+    metadata:
+      name: mongo-pod
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 5 but found 6

(indentation)